src/components/Sidebar.js

```diff
@@ -0,0 +1,32 @@
+
+export default function Sidebar({notes, currentNote, setCurrentNoteId, newNote, deleteNote}) {
+    const noteElements = notes.map((note, index) => (
+        <div key={note.id}>
+            <div
+                
+                className={`title ${
+                    note.id === currentNote.id ? "selected-note" : ""
```

Comment:
since you only have one alternative, you can make this  a logical AND operator 
```suggestion
                    note.id === currentNote.id && "selected-note" 
```